tests/unit/test_models/test_submodels/test_electrode/test_ohm/test_leading_size_distribution.py

```diff
@@ -0,0 +1,52 @@
+#
+# Test leading-order ohm submodel
```

Comment:
I don't think these single-submodel tests are necessary any more. In practice they don't really help to catch bugs and are just a pain to maintain (changing which variables are input)

Reply:
This test is for an old version of a submodel and forgot to remove it. I see what you mean about them just being a pain (had to fix several other tests myself by adding variables/options)